.github/workflows/ReactNativeSlider-CI.yml

```diff
@@ -109,29 +109,71 @@ jobs:
         with:
           path: |
             example/node_modules
-            example/ios/Pods
-          key: ${{ hashFiles('./example/package-lock.json') }}-${{ hashFiles('./package/package-lock.json') }}-${{ hashFiles('./example/ios/Podfile.lock') }}
+          key: ${{ hashFiles('./example/package-lock.json') }}-${{ hashFiles('./package/package-lock.json') }}
 
-      - name: Install required dependencies on cache miss
+      - name: Install required dependencies on cache miss (npm)
         if: steps.cache-npm.outputs.cache-hit != 'true'
         run: |
           npm install
-          cd example/ios && pod install --verbose
+
+      - name: Install pods
+        run: pod install --verbose
+        working-directory: example/ios
 
       - name: Use the current package sources in build
         run: cd example && npm run refresh-package
 
-      - name: Opening Simulator app
-        uses: futureware-tech/simulator-action@v1
+      - name: Build iOS
+        run: |
+          device_name='iPhone 13'
+          device=$(xcrun simctl list devices "${device_name}" available | grep "${device_name} (")
+          re='\(([-0-9A-Fa-f]+)\)'
+          [[ $device =~ $re ]] || exit 1
+          xcodebuild -workspace example.xcworkspace -scheme example -destination "platform=iOS Simulator,id=${BASH_REMATCH[1]}" CODE_SIGNING_ALLOWED=NO COMPILER_INDEX_STORE_ENABLE=NO build
```

Comment:
Just wondering: Don't we have `COMPILER_INDEX_STORE_ENABLED` set to `NO` by default?

Reply:
It defaults to YES, and it gives us major speed improvements. Check this out: 

- https://twitter.com/steipete/status/1100404024224804871
- https://stackoverflow.com/questions/71109718/what-is-the-compiler-index-store-enable-xcode-build-settings-default-value